app/src/main/java/net/kwatts/powtools/adapters/RideListAdapter.java

```diff
@@ -104,4 +117,18 @@ void bind(RideRow rideRow) {
         }
     }
 
+    void loadTrack(long rideId, @NonNull ImageView imageView) {
+        App.dbExecute(database -> {
+            List<Pair<Double, Double>> track = new ArrayList<>();
+            List<Moment> moments = database.momentDao().getFromRide(rideId);
+            for (Moment moment : moments) {
+                track.add(new Pair<>(moment.getGpsLongDouble(), moment.getGpsLatDouble()));
+            }
+
+            new Handler(Looper.getMainLooper()).post(() ->
+                    imageView.setImageDrawable(new TrackDrawable(track, 1000f, 0.2f))
```

Comment:
magic numbers, should probably use track bounds to compute. ;)

Reply:
Added named constant in `TrackDrawable` to make it less "magic".